internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStoreGraveyard.java

```diff
@@ -112,23 +138,45 @@ public void putStoreIntoGraveyard(String clusterName, Store store) {
             largestUsedVersionNumber);
         store.setLargestUsedVersionNumber(largestUsedVersionNumber);
       }
-    } else if (store.getLargestUsedVersionNumber() < largestUsedVersionNumber) {
-      // largestUsedVersion number in re-created store is smaller than the deleted store. It's should be a issue.
-      String errorMsg = "Invalid largestUsedVersionNumber: " + store.getLargestUsedVersionNumber() + " in Store: "
-          + store.getName() + ", it's smaller than one found in graveyard: " + largestUsedVersionNumber;
-      LOGGER.error(errorMsg);
-      throw new VeniceException(errorMsg);
+      if (largestUsedRTVersionNumber > store.getLargestUsedRTVersionNumber()) {
```

Comment:
Why do we need this check for RT during store migration? It doesn't make sense to me based on the above comments.  Am I missing something?